client/components/list-end/index.js

```diff
@@ -0,0 +1,8 @@
+/**
+ * External dependencies
+ */
+import React from 'react';
+
+export default function ListEnd() {
+	return <div className="list-end" />;
```

Comment:
Very nice! I updated to this in c186563. I added an eslint disable comment because `<div className="list-end" />;` was oddly provoking a lint `wpcalypso/jsx-classname-namespace` error.